src/business-logic/services/tv2-inews-ingest-service.ts

```diff
@@ -0,0 +1,23 @@
+import { IngestService } from './interfaces/ingest-service'
+import { HttpRequestParameters, HttpService } from './interfaces/http-service'
+import { RundownRepository } from '../../data-access/repositories/interfaces/rundown-repository'
+import { Rundown } from '../../model/entities/rundown'
+
+const DEFAULT_URL: string = 'localhost:3007'
```

Comment:
This is not a url. And since the env variables don't change after this point, you check it here, such that this becomes:
```typescript
const INEWS_HOST: string = process.env.INEWS_HOST ?? 'localhost:3007'
```

Reply:
I have changed the constant to fit this!